Examples/Oscilloscopes/TekSeriesScopes_HighSpeedDigitizers/src/SavePicLocalTimeExample/Save_Pic_test.py

```diff
@@ -0,0 +1,149 @@
+''' 
+Author: FAE team Americas and COE (US, AU)
+Python 3 64-bit 
+NumPy 1.11.2, MatPlotLib 2.0.0, PyVISA 1.11
```

Comment:
```suggestion
PyVISA 1.11
```
Remove since numpy and matplotlib aren't used.